cypress/support/authorityApi.js

```diff
@@ -0,0 +1,93 @@
+import Axios from 'axios';

```

Comment:
hvorfor trengs denne fila? tanken er vel å teste frontend-implementasjonen heller enn å ha en egen implementasjon her?
Nå har vi samme kode her som i `authorityApi.js` i frontend og to plasser for vedlikehold

Reply:
Trenger å opprette og manipulere autoriteter for å sette opp før-tilstand til testene.